samples/Program.cs

```diff
@@ -1,20 +1,28 @@
-﻿using System.IO;
+using System.Linq;
+using Microsoft.AspNetCore;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration.Json;
 
 namespace SodaPop.ConfigExplorer.Sample
 {
     public class Program
     {
         public static void Main(string[] args)
         {
-            var host = new WebHostBuilder()
-                .UseKestrel()
-                .UseContentRoot(Directory.GetCurrentDirectory())
-                .UseIISIntegration()
-                .UseStartup<Startup>()
-                .Build();
 
-            host.Run();
+            CreateWebHostBuilder(args).Build().Run();
         }
+
+        public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
+            WebHost.CreateDefaultBuilder(args)
+                .ConfigureAppConfiguration(c =>
+                {
+                    // Remove non-JSON configuration sources
+                    foreach (var source in c.Sources.Where(s => !(s is JsonConfigurationSource)).ToArray())
```

Comment:
Why this?

Reply:
Basically this:

> The output is very verbose and does make it difficult to parse when there is a lot of configuration. This is especially true since the default web host builder comes with environment variables, so there are a lot of entries without a hierarchy.

If you don't filter out the environment variables, the output explodes and just doesn't look that useful. I hide it here so that users can see what a normal hierarchy would look like.

In the end, we should come up with a better layout though that supports the amount of flat configuration values.